packages/blaze/template.js

```diff
@@ -267,6 +267,8 @@ Blaze.TemplateInstance = function (view) {
    * @type {DOMNode}
```

Comment:
This is the commit you want to split? https://github.com/meteor/meteor/commit/1f77d8e40ffca2f9b86e9fb5078b6c77487acb32